cmd/konvoy-image/cmd/args.go

```diff
@@ -7,7 +7,7 @@ import (
 )
 
 func addOverridesArg(fs *pflag.FlagSet, overrides *[]string) {
-	fs.StringArrayVar(overrides, "overrides", []string{}, "a list of override YAML files")
+	fs.StringSliceVar(overrides, "overrides", []string{}, "a comma separated list of override YAML files")
```

Comment:
So its still possible to set `--overrides=1.yaml --overrides=2.yaml` right? I saw the PR description but was a little confused if this meant only "a comma separated list"

Reply:
yes. it is still possible. however I was not sure better way to specify that in short one line in the flag description. So I just mentioned latest way to specify the flag values. any suggestion on the description change? or the current change in description is fine?